rotkehlchen/tests/unit/test_ethereum_airdrops.py

```diff
@@ -67,41 +139,58 @@ def test_check_airdrops(ethereum_accounts, database, freezer):
             location_label=string_to_evm_address(TEST_ADDR1),
         ),
     ]
+    MOCK_AIRDROP_INDEX['airdrops']['shutter']['new_asset_data'] = new_asset_data
+
+    # remove new asset from globalDB and cache
+    new_asset_identifier = MOCK_AIRDROP_INDEX['airdrops']['shutter']['asset_identifier']
+    for globaldb_conn in (GlobalDBHandler().conn, GlobalDBHandler().packaged_db_conn()):
+        with globaldb_conn.write_ctx() as write_cursor:
```

Comment:
mmmmm the problem of this approach is that you modify the packaged globaldb and this is bad when doing the tests. Can you maybe mock the AssetResolver to not query the packaged db_conn or that it returns no assets when queried for that asset the first time? Ask in discord if you have trouble with it but you shouldn't modify the packaged global db